### PR TITLE
Feat/met 1762 remove mocked periodic loop from executor

### PR DESCRIPTION
**Pull request by stzanakis**

### CODE REVIEW

File: metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/WorkflowExecutor.java [hunk 1]
```diff
@@ -364,37 +359,4 @@ private void preparePluginStateAndFinishedDate(AbstractMetisPlugin abstractMetis
...
-    }
-    abstractMetisPlugin.setFinishedDate(new Date());
-    abstractMetisPlugin.setPluginStatusAndResetFailMessage(PluginStatus.FINISHED);
-    workflowExecutionDao.updateWorkflowPlugins(workflowExecution);
-  }
-
-  private void fakeMonitorUpdateProcessedRecords(AbstractMetisPlugin abstractMetisPlugin,
-      int iteration) {
-    ExecutionProgress executionProgress = abstractMetisPlugin.getExecutionProgress();
-    executionProgress.setExpectedRecords(MONITOR_ITERATIONS_TO_FAKE * FAKE_RECORDS_PER_ITERATION);
```
Comment (jochen-vermeulen) [inline metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/WorkflowExecutor.java]: You have not removed the two constants: they are no longer needed.

File: metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/AbstractMetisPlugin.java [hunk 2]
```diff
@@ -365,18 +365,16 @@ DpsTask createDpsTaskForIndexPlugin(EcloudBasePluginParameters ecloudBasePluginP
...
-          ecloudBasePluginParameters.getEcloudDatasetId());
-      try {
-        DpsTask dpsTask = prepareDpsTask(ecloudBasePluginParameters);
-        setExternalTaskId(Long.toString(dpsClient.submitTask(dpsTask, getTopologyName())));
-      } catch (DpsException | RuntimeException e) {
-        throw new ExternalTaskException("Submitting task failed", e);
-      }
-      LOGGER.info("Submitted task with externalTaskId: {}", getExternalTaskId());
+    String pluginTypeName = getPluginType().name();
+    LOGGER.info("Starting real execution of {} plugin for ecloudDatasetId {}", pluginTypeName,
```
Comment (jochen-vermeulen) [inline metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/AbstractMetisPlugin.java]: Remove the word 'real'?

File: metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/WorkflowExecutor.java (hunk 1 above)
Comment (stzanakis) [reply metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/WorkflowExecutor.java]: Fixed

File: metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/AbstractMetisPlugin.java (hunk 2 above)
Comment (stzanakis) [reply metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/AbstractMetisPlugin.java]: Fixed

**Pull request merged**